src/Banner.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Box from './Box'
+import Flex from './Flex'
+import Text from './Text'
+import Icon from './Icon'
+import CloseButton from './CloseButton'
+import Heading from './Heading'
+import { palette } from './theme'
+import PropTypes from 'prop-types'
+
+const Banner = (props) => {
+  const icon = palette[props.palette].icon
+
+  return (
+    <Flex
+      {...props}
+      bg={palette[props.palette].backgroundColor}
+      color={palette[props.palette].color}
+      justify='space-between'
+      align='center'
+    >
+      {!!icon && !!props.showIcon && <Icon name={icon} mr={3} size={28} />}
```

Comment:
Why both `icon` and `showIcon`? Seems like checking for `icon` should be okay

Reply:
Oh, nm. I see that `icon` isn't coming from props... Mentioned this earlier, but think it would be good to keep the icon configurable via props, rather than tied to some semantic meaning. It would make it less automatic, but more flexible down the road

Reply:
What would you think about `const icon = props.icon || palette[props.palette].icon`?  Would that be the best of both worlds in terms of having presets and configurations?